doc/articles/uno-fluent-assets.md

```diff
@@ -15,14 +15,27 @@ The font is in this repository https://github.com/unoplatform/uno.fonts
 
 ## Changes
 
-| Device family | Description   |
-| -- | -- |
-| iOS & macOS | Once Uno has been updated, it will start looking for a font named symbols (Not necessarily the name of the file). For this font to be available it needs to be in the resource folder.  ![image](Assets/font-ios.png) ![image](Assets/font-macos.png)  The `info.plist` should also be updated for both platforms, replacing winjs-symbols.ttf by uno-fluentui-assets.ttf|
-| Android | Once Uno has been updated, it will start looking for a font file named uno-fluentui-assets.ttf in its assets folder:  ![image](Assets/font-android.png) |
-| WebAssembly | A wasm build won't break after the update but to access the new symbols the file Font.css should be chnaged. The font is passed as a base64 file. ![image](Assets/font-wasm.png) |
+### iOS & macOS 
+---
+The `info.plist` should be updated for both platforms, replacing winjs-symbols.ttf by uno-fluentui-assets.ttf
+
+Uno on those plateform is looking for a font named symbols (A fonts name is not necessarily the name of the file). For this font to be available it needs to be in the resource folder.  \
+![image](Assets/font-ios.png) 
+![image](Assets/font-macos.png)  
+
+
+### Android 
+---
+Once Uno has been updated, it will start looking for a font file named uno-fluentui-assets.ttf in the assets folder: \
+![image](Assets/font-droid.png)
+
+### WebAssembly 
+---
+A wasm build won't break after the update but to access the new symbols the file Font.css should be changed. The font is passed as a base64 file: \
+![image](Assets/font-wasm.png) 
 
 ## Limitations
-On iOS and macOS the indeterminate state for a check box is not the right color.
+On iOS and macOS the indeterminate state for a CheckBox is not the right color.
 
 
 ## Related Topics
```

Comment:
For the related topics - one of the items starts with `#`, one does not, maybe remove it so it's the same for both items. OCD 😂

Reply:
Ahah, I get it, I cannot believe I wrote it like that.